docs/source/tutorials.rst

```diff
@@ -0,0 +1,82 @@
+Tutorials
+=========
+
+Here, we provide a small set of tutorials to help ypu get started with lwreg. 
+Please keep in mind that lwreg allow for problem dependent customization:
+The tutorials are meant as a starting point, but might not be applicable to your specific use case.
+
+Setting up a Database and First Registration
+---------------------------------------------
+For a single user, SQLite is the easiest way to get started with lwreg.
+In this first example, we will show how to set up an lwreg instance for registering molecules and querying them.
+We will also see how to retrieve the registerd structures::
+     
+  import lwreg
+  from lwreg import utils
+
+  # we need to start with a config, here let's use the default provided by lwreg
+  config = utils.defaultConfig()
+
+  # the default config is the following:
+  # config = {'dbname': './testdb.sqlt',
+  # 'dbtype': 'sqlite3',
+  # 'standardization': 'fragment',
+  # 'removeHs': 1,
+  # 'useTautomerHashv2': 0,
+  # 'registerConformers': 0,
+  # 'numConformerDigits': 3,
+  # 'lwregSchema': ''}
+
+  # now we can initialize the database
+  lwreg.initdb(config)
+
+  # after confirming the intialization, we can register some molecules
+  from rdkit import Chem
+  from rdkit.Chem import Draw
+  from rdkit.Chem.Draw import IPythonConsole
+  
+  smis = ["Cc1[nH]ncc1[C@H](F)Cl","Cc1[nH]ncc1[C@@H](F)Cl","Cc1[nH]ncc1[CH](F)Cl","Cc1n[nH]cc1[C@@H](F)Cl"]
+  ms = [Chem.MolFromSmiles(smi) for smi in smis]
+  IPythonConsole.drawOptions.useBWAtomPalette()
+  IPythonConsole.drawOptions.legendFontSize=24
+  Draw.MolsToGridImage(ms,legends=[str(i+1) for i in range(len(ms))],molsPerRow=2,subImgSize=(300,300))
```

Comment:
I know we've talked about this, but I will be tiresome and mention it again: this is notebook code and not useful in a standard python script. I think it only makes sense to include it in the context of a notebook